src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineView.scala

```diff
@@ -30,6 +30,11 @@ case class PipelineView(name: String,
             pubStatementSB.append(s"and ${pCol._1} = ${dataSource.name}.${pCol._2} ")
           })
         }
+        if (workspacesAllowed.length !=0){
+          val workspacesAllowedString = workspacesAllowed.mkString(",")
+          pubStatementSB.append(s" and organization_id in (${workspacesAllowedString})")
+          println("pubStatementSB after workspacesAllowedString is ",pubStatementSB)
```

Comment:
printing out something called `pubStatementSB` won't make sense to the user. Did you mean to print the variable or was this debug? We shouldn't write out the variable name here but rather something more descriptive.